### PR TITLE
Support searching entries with fields inside repeater field

### PR DESCRIPTION
**Pull request by AbdiTolesa**
Fixes https://github.com/Strategy11/formidable-pro/issues/3843
I exposed fields inside repeater to appear in the entries search box in pro (https://github.com/Strategy11/formidable-pro/pull/4061) and that didn't cover it all as searching with those new fields wasn't supported. This update adds support for searching using fields embedded in a repeater.

### CODE REVIEW

File: classes/helpers/FrmEntriesListHelper.php [hunk 1]
```diff
@@ -13,16 +13,31 @@ class FrmEntriesListHelper extends FrmListHelper {
...
+
 	public function prepare_items() {
 		global $per_page;
 
 		$per_page = $this->get_items_per_page( 'formidable_page_formidable_entries_per_page' );
 		$form_id  = $this->params['form'];
 
 		$s_query = array();
-
+		$form_ids = $this->get_repeater_form_ids( $form_id );
```
Comment (Crabcyborg) [inline classes/helpers/FrmEntriesListHelper.php:37]: Similar to the other branch, what do you think about that `searchable_form_ids` function? Does this work with embedded fields yet, and can we support those too if it doesn't?

File: classes/helpers/FrmEntriesListHelper.php [hunk 2]
```diff
@@ -13,16 +13,31 @@ class FrmEntriesListHelper extends FrmListHelper {
 	 */
 	public $total_items = 0;
 
+	private function get_repeater_form_ids( $form_id ) {
```
[the remark this answers is not in the document]
Comment (AbdiTolesa) [reply classes/helpers/FrmEntriesListHelper.php:16]: @Crabcyborg I was thinking the same but if we reused this function from lite and removing it from PRO, what if PRO is updated but not lite then?

[the remark this answers is not in the document]
Comment (Crabcyborg) [reply classes/helpers/FrmEntriesListHelper.php:16]: @AbdiTolesa It really should only be in Pro. This isn't a great place for it anyway. Ideally we don't have add any new references for repeaters here into Lite.

[the remark this answers is not in the document]
Comment (Crabcyborg) [reply classes/helpers/FrmEntriesListHelper.php:16]: I didn't really notice at first this update was in Lite. Can we avoid updating Lite for this? We have a `FrmProEntriesListHelper` that inherits this helper. Can we just overwrite the `prepare_items` function there?

[the remark this answers is not in the document]
Comment (AbdiTolesa) [reply classes/helpers/FrmEntriesListHelper.php:16]: @Crabcyborg Yeah sure, I should have thought of that! I just moved all of these updates to pro and I think it is safe to close this.

**Pull request closed without merging**